examples/cross-contract-calls/low-level/Cargo.toml

```diff
@@ -3,9 +3,27 @@ name = "cross-contract-low-level"
 version = "1.1.0"
 authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2021"
+repository = "https://github.com/near/near-sdk-rs"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
 near-sdk = { path = "../../../near-sdk" , features = ["default", "unit-testing"] }
+
+[package.metadata.near.build]
+# This is the Docker metadata line:
+image = "nearprotocol/contract-builder:latest-amd64"
+
+[package.metadata.near.reproducible_build]
+# docker image, descriptor of build environment
+image = "nearprotocol/contract-builder:latest-amd64"
+# tag after colon above serves only descriptive purpose; image is identified by digest
+image_digest = "sha256:53b34ea28b8995c6e8908e76db8ea0b117ac73bc91c83c8aae930e635e9ef885"
```

Comment:
replace all of the images with https://hub.docker.com/layers/sourcescan/cargo-near/0.13.5-rust-1.85.1/images/sha256-3b0272ecdbb91465f3e7348330d7f2d031d27901f26fb25b4eaf1560a60c20f3 ?